polarization/tissueLayer.py

```diff
@@ -50,6 +50,11 @@ def transferMatrix(self, dz=None) -> BirefringentMaterial:
             dz = self.thickness
         return BirefringentMaterial(deltaIndex=self.birefringence, fastAxisOrientation=self.orientation, physicalLength=dz)
 
+    def backscatterTransferMatrix(self, k):
```

Comment:
I think this should not really be done explicitly like that, it should be a new class such as "MultiBirefringentMaterial" or something like that.  We can deal with this later.